ods_ci/tests/Resources/Files/llm/serving_runtimes/vllm_servingruntime_http.yaml

```diff
@@ -12,7 +12,7 @@ spec:
         - '--served-model-name={{.Name}}'
         - '--distributed-executor-backend=mp'
         - '--chat-template=/app/data/template/template_chatml.jinja'
-      image: ${runtime_image}
+      image: quay.io/modh/vllm@sha256:167aa05982214c3941afbdde7c5aff6f81040193a0db1dc06a5ad16af0a8780e
```

Comment:
Can you explain about this image change ?

Reply:
ignore, rebasing